src/main.scss

```diff
@@ -0,0 +1,19 @@
+@import "./styles/var/var.scss";
+
+@import "./styles/container.scss";
+@import "./styles/header.scss";
+@import "./styles/logo.scss";
+@import "./styles/nav.scss";
+@import "./styles/is-active.scss";
+@import "./styles/card.scss";
+@import "./styles/starts.scss";
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
```

Comment:
Don't use * selector for zeroing out your margins or paddings. It's still inefficient for browser to read your web document